src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+                                Transferred ->
+                                    t "transfer_result.transfer_success"
+
+                                Received ->
+                                    t "transfer_result.receive_success"
+
+                                NotInvolved ->
+                                    t "transfer_result.transfer_success"
+                        ]
+                    ]
+                , div [ class "bg-no-repeat bg-auto h-64 ml-32 -mt-5 px-4 py-2 m-2 items-center mt-5 justify-center", style "background-image" "url(/images/transfer-doggo.svg)" ]
+                    []
+                , div [ class "flex sm:justify-center sm:items-center -mt-16 absolute md:w-full md:mx-auto  sm:right-0" ]
+                    [ viewTransferCard loggedIn transfer state
+                    ]
+                ]
+            ]
+        ]
+
+
+viewTransferCard : LoggedIn.Model -> Transfer -> State -> Html Msg
+viewTransferCard loggedIn transfer state =
+    let
+        ipfsUrl =
+            loggedIn.shared.endpoints.ipfs
+    in
+    div [ class "flex flex-row block rounded overflow-x-scroll sm:overflow-hidden bg-gray-100" ]
+        [ div [ class "sm:px-4 sm:py-2 sm:m-2 p-0" ]
+            [ div [ class "h-8 w-8 rounded-full mx-auto mt-5" ]
+                [ Avatar.view ipfsUrl
+                    (case state of
+                        Received ->
+                            transfer.to.avatar
+
+                        Transferred ->
+                            transfer.from.avatar
+
+                        NotInvolved ->
+                            transfer.from.avatar
+                    )
+                    "h-10 w-10"
+                ]
+            , div [ class "px-6 py-4" ]
+                [ span [ class "text-base inline-block bg-black rounded-full px-3 py-1 text-sm font-semibold text-white" ]
+                    [ case state of
+                        Received ->
+                            text "You"
+
+                        Transferred ->
+                            text "You"
+
+                        NotInvolved ->
+                            Eos.viewName transfer.fromId
+                    ]
+                ]
+            ]
+        , div [ class "mt-5" ] [ viewAmount loggedIn transfer state ]
+        , div [ class "-ml-20 sm:px-4 sm:py-2 m-2" ]
+            [ div [ class "h-8 w-8 rounded-full mx-auto mt-5" ]
+                [ Avatar.view ipfsUrl
+                    (case state of
+                        Received ->
+                            transfer.from.avatar
+
+                        Transferred ->
+                            transfer.to.avatar
+
+                        NotInvolved ->
+                            transfer.to.avatar
+                    )
+                    "h-10 w-10"
+                ]
+            , div [ class "px-6 py-4" ]
+                [ span [ class "inline-block bg-black rounded-full px-3 py-1 text-sm font-semibold text-white" ]
```

Comment:
Both of the account pills should be different. They aren't `rounded-full`. Check print from Figma and comparing to the implementation:

<img width="149" alt="Screen Shot 2019-12-19 at 13 09 04" src="https://user-images.githubusercontent.com/1082127/71190900-25888780-2264-11ea-9dda-d9c5d0f6c641.png">
<img width="131" alt="Screen Shot 2019-12-19 at 13 09 09" src="https://user-images.githubusercontent.com/1082127/71190902-25888780-2264-11ea-98c3-dea6c4d86727.png">


Also the string should be in uppercase. There is a class in Tailwind that does that